tools/version2sha.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright(c) 2021 Intel Corporation
+# Copyright(c) 2021-2022 Intel Corporation
```

Comment:
I think we should not update the copyright date in a files where the content doesn't change

Reply:
Fixed with this PR: https://github.com/Open-CAS/open-cas-linux/pull/1340